packages/catalog-process/src/utilities/errorMappers.ts

```diff
@@ -36,6 +36,7 @@ export const deleteEServiceErrorMapper = (
   match(error.code)
     .with("eServiceNotFound", () => HTTP_STATUS_NOT_FOUND)
     .with("operationForbidden", () => HTTP_STATUS_FORBIDDEN)
+    .with("eserviceCannotBeUpdatedOrDeleted", () => HTTP_STATUS_BAD_REQUEST)
```

Comment:
This mapper is used in the `deleteEService` operation. The error was thrown but not handled.